chia/server/class_for_type.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
```

Comment:
Happy to change the name. You probably have a much more complete view as to the abstract purpose of this and possibly a concept as to what's really going on here, so I'll defer to your suggestions.

Reply:
In so much as `ApiProtocol` is a "good" name for the interface, I guess just `apis.py`?  Since it is a place that would sensibly hold a list of all of them if we needed that and so on.  It is defining our group of APIs.  Maybe it also belongs directly under `chia/` since that would be the common root of all the imports.  Not that that has been identified as a hard rule to apply, but it is something I think about.